src/Xamarin.Android.Build.Tasks/Tasks/GetAdditionalResourcesFromAssemblies.cs

```diff
@@ -71,8 +70,14 @@ string SubstituteEnvVariables (string path)
 			MatchEvaluator replaceEnvVar = (Match m) => {
 				var e = m.Groups [2].Value;
 				switch (e) {
-				case "ANDROID_SDK_PATH": return AndroidSdkDirectory.TrimEnd (Path.DirectorySeparatorChar);
-				case "ANDROID_NDK_PATH": return AndroidNdkDirectory.TrimEnd (Path.DirectorySeparatorChar);
+				case "ANDROID_SDK_PATH":
+					return AndroidSdkDirectory.TrimEnd (Path.DirectorySeparatorChar);
+				case "ANDROID_NDK_PATH":
+					//NOTE: AndroidNdkDirectory is not [Required]
```

Comment:
@jonathanpeppers this code path is only used for `Android.IncludeAndroidResourcesFromAttribute`, `Android.NativeLibraryReferenceAttribute` and `Java.Interop.JavaLibraryReferenceAttribute` entries which are normally included in one of the Support Libraries. So we might have an attribute like

```
[assembly: Android.NativeLibraryReference ("ANDROID_NDK_PATH/platforms/android-29/arch-arm64/usr/lib/libEGL.so")]
```

 Now I'm not sure if there are any of our support libraries which do use that but if there are and `AndroidNdkDirectory` is blank we might have a problem with the build later. We should probably add a unit test to make sure this works and/or raises a decent error message if `AndroidNdkDirectory` is blank and we hit one of these.